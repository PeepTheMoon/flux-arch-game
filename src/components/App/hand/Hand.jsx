import React from 'react';
import PropTypes from 'prop-types';

const Hand = ({ image }) => (
  <figure>
    <img src={image} />
  </figure>
);

Hand.propTypes = {
  image: PropTypes.string.isRequired
};

export default Hand;
