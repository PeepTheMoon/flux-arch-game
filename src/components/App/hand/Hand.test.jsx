import React from 'react';
import { shallow } from 'enzyme';
import Hand from './Hand';

describe('Hand component', () => {
  it('renders Hand', () => {
    const wrapper = shallow(<Hand image="https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcRolJqqHMjpTKMX1xJ5ytee8332VT_vLI4PoAsurW4zcQQrE8Nd&usqp=CAU" />);
    expect(wrapper).toMatchSnapshot();
  });
});
