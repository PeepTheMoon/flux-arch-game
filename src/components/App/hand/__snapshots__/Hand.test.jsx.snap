// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hand component renders Hand 1`] = `
<figure>
  <img
    src="https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcRolJqqHMjpTKMX1xJ5ytee8332VT_vLI4PoAsurW4zcQQrE8Nd&usqp=CAU"
  />
</figure>
`;
